flipperdevices/Flipper-Android-App
# Improve file edit screen

**Background**

File editor currently contains a lot of logic, which makes code reading harder. It can be improved by splitting it into separate decompose screen for each task: Loading, Editing, Uploading

**Changes**

- Split editor into multiple decompose screens

**Test plan**

- Open new editor. See new beautiful new loading screen with progress.
- After it loaded try edit your file
- After finish try save file or save file as new file.
- See new file immediately appeared in listing with updated size
